src/rules/signifyd-score.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @title Detect Ecommerce Fraud Users 
+ * @overview Get the signifyd score from signfyd.com and store it on app_metadata.
+ * @gallery true
+ * @category enrich profile
+ *
+ *
+ * This rule gets the signifyd score and status from signifyd.com and stored it in app_metadata.
+ *
+ */
+
+function getSignifydScore(user, context, callback) {
+  
+    user.app_metadata = user.app_metadata || {};
+    if (!user.app_metadata.caseId) return callback(null, user, context);
+  
+    if(!user.email) {
+      // the profile doesn't have email so we can't query their api.
+      return callback(null, user, context);
+    }
+  
+    const request = require('request');
+  
+    request({
+      url: `https://api.signifyd.com/v2/cases/${user.app_metadata.caseId}`,
+      headers: {
+          "Authorization": "Basic YWJjZGVmZ2hpamtsbW5vcHFyc3R1dnd4eXo6"
+        }
+    
+    }, function (err, resp, body) {
+      if (err) return callback(null, user, context);
+      if (resp.statusCode !== 200) return callback(null, user, context);
+      const signifyd = JSON.parse(body);
+      if (signifyd.status !== 'Ok') return callback(null, user, context);
+  
+      user.app_metadata = user.app_metadata || {};
+      user.app_metadata.guaranteeEligible = signifyd.data.status;
+      user.app_metadata.signifyd = signifyd.data.score;
```

Comment:
Should this be `user.app_metadata.signifyd_score = signifyd.data.score` instead?

Reply:
Yes, good catch